tests/src/FunctionalJavascript/ThunderMetaTagTrait.php

```diff
@@ -44,7 +44,12 @@ public function checkMetaTags(array $metaTagConfiguration) {
     foreach ($metaTagConfiguration as $metaTagName => $value) {
       $metaTag = explode(' ', $metaTagName);
 
-      $this->checkMetaTag($metaTag[1], $value);
+      if ($metaTag[1] == 'title') {
+        $this->assertSession()->elementContains('xpath', '//title', $value);
```

Comment:
Can we move this logic inside `checkMetaTag`?

Reply:
We should test for title independent of the other meta information. This way we can have the sam,e test for all versions of metatag module  and do not have to increase the min version, just for our tests

Reply:
I wouldn't do that. We check the title tag now and not a meta tag anymore